demo/kubernetes/README.md

```diff
@@ -0,0 +1,55 @@
+# Rook on Kubernetes
+
+A demo of running the rook storage system within a [Kubernetes](http://kubernetes.io/) cluster. We will use a mysql service in Kubernetes as an example of using block storage from rook.
```

Comment:
maybe add the word "persistent" here to emphasize the benefit of it? e.g. "using persistent block storage from rook"